### PR TITLE
PR: Move to github actions

### LINKED ISSUE #48: Move CI to github actions

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #48

### CODE REVIEW

**Comment by goanpeca** [comment]
Any comments @danbradham ?

File: qtsass/__init__.py [hunk 1]
```diff
@@ -41,5 +41,21 @@
 enable_logging()
 
 # Constants
-VERSION_INFO = (0, 1, 2, 'dev0')
-__version__ = '.'.join(map(str, VERSION_INFO))
+__version__ = '0.2.2.dev0'
```
Comment (ccordoba12) [inline qtsass/__init__.py]: This should be `0.2.0`, right?

Comment (goanpeca) [reply qtsass/__init__.py]: The latest tag on this repo us `0.2.2`also, it seems we never published to pypi (there is still 0.1.1) 😆

Comment (ccordoba12) [reply qtsass/__init__.py]: Ok, I see.

Comment (goanpeca) [reply qtsass/__init__.py]: So good to go?

**Pull request merged**